tests/scripts/deploy_admission_controller.sh

```diff
@@ -81,39 +81,4 @@ EOF
 
 echo "Successfully deployed cert-manager"
 
-echo "Creating Issuer and Certificate"
-cat <<EOF | kubectl create -f -
-apiVersion: cert-manager.io/v1
-kind: Issuer
-metadata:
-  name: selfsigned-issuer
-  namespace: ${NAMESPACE}
-spec:
-  selfSigned: {}
----
-apiVersion: cert-manager.io/v1
-kind: Certificate
-metadata:
-  name: rook-admission-controller-cert
-  namespace: ${NAMESPACE}
-spec:
-  dnsNames:
-  - ${SERVICE_NAME}
-  - ${SERVICE_NAME}.${NAMESPACE}.svc
-  - ${SERVICE_NAME}.${NAMESPACE}.svc.cluster.local
-  issuerRef:
-    kind: Issuer
-    name: selfsigned-issuer
-  secretName: rook-ceph-admission-controller
-EOF
-
-echo "Successfully created Issuer and Certificate"
-
-echo "Deploying webhook config"
-< "${BASE_DIR}"/webhook-config.yaml \
-        "${BASE_DIR}"/webhook-patch-ca-bundle.sh | \
-        sed -e "s|\${NAMESPACE}|${NAMESPACE}|g" | \
-        sed -e "s|\${WEBHOOK_CONFIG_NAME}|${WEBHOOK_CONFIG_NAME}|g" | \
-        sed -e "s|\${SERVICE_NAME}|${SERVICE_NAME}|g" | \
-        kubectl create -f -
-echo "Webhook deployed! Please start the rook operator to create the service and admission controller pods"
+echo "Please deploying webhook config and start the rook operator to create the service and admission controller pods"
```

Comment:
```suggestion
echo "Please deploy the webhook config and start the rook operator to create the service and admission controller pods"
```

Reply:
Why is this telling them to deploy the webhook config? Isn't that the point of this script already? Maybe it's just the same question as @leseb about why it's being removed if more steps are still needed.